src/Java.Interop.Localization/Resources.resx

```diff
@@ -332,4 +332,54 @@ Name, com.example.MyClass.</comment>
     <value>Cannot override Kotlin-generated method '{0}' because it is not a valid Java method name. This method can only be overridden from Kotlin.</value>
     <comment>{0} - Kotlin method name.</comment>
   </data>
+  <data name="JniMarshalMethodGen_JM4001" xml:space="preserve">
+    <value>Unable to preload reference '{0}'.</value>
+    <comment>{0} - assembly path</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM4002" xml:space="preserve">
+    <value>Please specify at least one ASSEMBLY to process.</value>
+    <comment>The following terms should not be translated or have any capitalization changes: ASSEMBLY. This is a special case for this particular message. In most messages, "assembly" would be translated.</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM4003" xml:space="preserve">
+    <value>Unable to create Java VM{0}{1}</value>
+    <comment>{0} - newline, {1} - exception</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM4004" xml:space="preserve">
+    <value>Unable to read profile file '{0}'.{1}{2}</value>
+    <comment>{0} - path, {1} - newline, {2} - exception. In this message, the term "profile" refers to a customized list of types to process.</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM4005" xml:space="preserve">
+    <value>Path '{0}' does not exist.</value>
+    <comment>{0} - path</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM4006" xml:space="preserve">
+    <value>Unable to process assembly '{0}'{1}{2}{1}{3}</value>
+    <comment>{0} - assembly, {1} - newline, {2} - exception message, {3} exception</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM8001" xml:space="preserve">
+    <value>Couln't find interface {0}</value>
+    <comment>{0} - interface name</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM8003" xml:space="preserve">
+    <value>Caught an exception while loading types. The types which cannot be loaded will not be processed. Make sure all referenced assemblies are available, use -r option. For details, check the exception:{0}{1}</value>
+    <comment>{0} - newline, {1} - exception</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM8004" xml:space="preserve">
+    <value>Unable to find type '{0}'. The type will not be processed. Make sure the paths to all reference assemblies are provided with the -L option.</value>
```

Comment:
Looking at this with fresh eyes again today, I realized that "reference assembly" is probably a little confusing because of [reference assemblies](https://docs.microsoft.com/dotnet/standard/assembly/reference-assemblies), so maybe saying "referenced assemblies" instead will avoid some confusion. And now that I see there's a `-r` option, maybe switching "paths" to "directories" for the `-L` option will also be handy to add while we're at it:

```diff
-<value>Unable to find type '{0}'. The type will not be processed. Make sure the paths to all reference assemblies are provided with the -L option.</value>
+<value>Unable to find type '{0}'. The type will not be processed. Make sure the directories for all referenced assemblies are provided with the -L option.</value>
```